inspire/modules/deposit/tasks.py

```diff
@@ -56,21 +64,23 @@ def arxiv_fft_get(obj, eng):
             dir=CFG_TMPSHAREDDIR,
         )
 
-        os.write(arxiv_file, arxiv_pdf.content)
-        os.close(arxiv_file)
```

Comment:
@pedrogaudencio I think you still need to call `os.close` to properly close the `arxiv_file` descriptor. See https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp